Engine/EditorUI/Panel/PanelProjectExplorer.cpp

```diff
@@ -207,9 +210,12 @@ void PanelProjectExplorer::ResourceDragSource(Metafile* metafile) const
 
 void PanelProjectExplorer::ProcessResourceMouseInput(Path* metafile_path, Metafile* metafile)
 {
-	if (ImGui::IsWindowHovered() && ImGui::IsMouseClicked(0))
+	if (ImGui::IsWindowHovered() && ImGui::IsMouseReleased(0))
```

Comment:
Why using ImGui::IsMouseReleased here and GetMouseButtonUp(MouseButton::Left) on PanelHerarchy? 
I get that the difference is that you catch if is the left click, but why not use the mousereleased in the panel herarchy?

Reply:
I'm using MouseRelease here to not get confuse with the drag and drop (MouseClick) son in case you have a gameobejct selected and you start a drag and drop it doesn't change the panel